### PR TITLE
refactor: remove connector enabled state in database

### PR DESCRIPTION
**Pull request by wangsijie**
## Summary

remove connector's `enabled` from database: now if a row exists, the connector is "enabled".

and use "add" and "delete" to replace "enable" and "disable".

for console, use `/api/connector-factories` to show the full list.

## Testing

UTs and local tested.

### CODE REVIEW

File: packages/console/src/pages/Connectors/components/CreateForm/index.tsx [hunk 1]
```diff
@@ -21,25 +24,44 @@ type Props = {
...
+
+    const allGroups = getConnectorGroups<ConnectorFactoryResponse>(
+      factories.filter(({ type: factoryType }) => factoryType === type)
+    );
+
+    return allGroups.map((group) => ({
+      ...group,
+      connectors: group.connectors.map((connector) => ({
+        ...connector,
+        added: connectors.some(({ connectorId }) => connector.id === connectorId),
```
Comment (simeng-li) [inline packages/console/src/pages/Connectors/components/CreateForm/index.tsx:52]: nit: might be a little bit confusing between factory group connectors and DB connectors here.

File: packages/console/src/pages/Connectors/components/SignInExperienceSetupNotice/index.tsx [hunk 2]
```diff
@@ -16,13 +16,7 @@ const SignInExperienceSetupNotice = () => {
...
   } = useUserPreferences();
 
-  if (!connectors || connectorSieNoticeConfirmed) {
-    return null;
-  }
-
-  const hasSetupConnector = connectors.some(({ enabled }) => enabled);
-
-  if (!hasSetupConnector) {
+  if (!connectors || connectors.length > 0 || connectorSieNoticeConfirmed) {
```
[the remark this answers is not in the document]
Comment (wangsijie) [reply packages/console/src/pages/Connectors/components/SignInExperienceSetupNotice/index.tsx:19]: When there is at least one connector, this notice is no longer needed, so `connectors.length === 0` is confirmed to be OK.

File: packages/console/src/pages/Connectors/components/CreateForm/index.tsx (hunk 1 above)
Comment (wangsijie) [reply packages/console/src/pages/Connectors/components/CreateForm/index.tsx]: updated with a new name, PTAL

**Review by simeng-li — APPROVED** [review]
LGTM

**Pull request merged**